src/main/java/org/apache/commons/lang3/builder/ReflectionToStringBuilder.java

```diff
@@ -867,4 +874,21 @@ private void validate() {
         }
     }
 
+    private boolean handleNativeClasses() {
+        Object value = getObject();
+        if (value instanceof Number || value instanceof Boolean || value instanceof Character
+                || value instanceof TemporalAccessor) {
```

Comment:
This expression uses `TemporalAccessor` here but the new `isJavaInternalClass()` uses `Temporal`. `Temporal` extends `TemporalAccessor`, why is there a difference? It seems this expression could be rewritten to call  `isJavaInternalClass()`, this centralize the types to check.